bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -8,11 +8,11 @@ metadata:
           "apiVersion": "rhpam.kiegroup.org/v1",
           "kind": "KogitoBuild",
           "metadata": {
-            "name": "process-quarkus-example"
+            "name": "ruleunit-quarkus-example"
```

Comment:
Is this replace intentional?

Reply:
I am suprised why these changes are missing in first place itself. Examples are already renamed to `ruleunit`.https://github.com/kiegroup/kogito-operator/blob/e93ee6989c548a32e26959fae6d22f7acc75e923/config/samples/rhpam/rhpam_v1_kogitobuild.yaml#L4

Reply:
For now I just run `make vet` cmd and it generate me a new CSV file.